src/transformers/models/swin/modeling_swin.py

```diff
@@ -1298,7 +1298,6 @@ def forward(
         >>> outputs = model(**inputs)
         >>> feature_maps = outputs.feature_maps
         >>> list(feature_maps[-1].shape)
-        [1, 768, 7, 7]
```

Comment:
Why remove this one?

Reply:
@alaradirik this looks like it was removed by mistake in yesterday's commit. Reverting this change. Hope that's alright?

Reply:
@shivalikasingh95 yes, it was removed by mistake indeed!